tests/test_attribution_service.py

```
@@ -0,0 +1,115 @@
+            assert size == 1024
+
+    @pytest.mark.asyncio
+    async def test_validate_file_size_too_large(self):
+        """Test file size validation for oversized file"""
+        mock_file = AsyncMock(spec=UploadFile)
+        mock_file.read.return_value = b"x" * (11 * 1024 * 1024)  # 11MB file
+
+        with patch(
+            "app.core.config.settings.MAX_FILE_SIZE", 10 * 1024 * 1024
+        ):  # 10MB limit
+            with pytest.raises(Exception):  # Should raise HTTPException
```

Ruff (0.13.2)

Do not assert blind exception: `Exception`

(B017)